File: C++/Najszybszy.cpp
#include <vector>
#include <iostream>
#include <algorithm>

int main() 
{
    std::vector<int> times(3);
    for (int &t : times) 
        std::cin >> t;
    int min_time = *std::min_element(times.begin(), times.end());
    if (std::count(times.begin(), times.end(), min_time) == 1)
        std::cout << "TAK " << (std::find(times.begin(), times.end(), min_time) - times.begin() + 1);
    else
        std::cout << "NIE";
    return 0;
}

File: Java/Najszybszy.java
import java.util.Arrays;
import java.util.Scanner;

public class Main 
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);
        int[] times = {input.nextInt(), input.nextInt(), input.nextInt()};
        int minTime = Arrays.stream(times).min().getAsInt();
        if (Arrays.stream(times).filter(t -> t == minTime).count() == 1)
            System.out.println("TAK " + (Arrays.asList(times[0], times[1], times[2]).indexOf(minTime) + 1));
        else
            System.out.println("NIE");
        input.close();
    }
}